pkg/minikube/cni/kindnet.go

```diff
@@ -55,6 +57,14 @@ rules:
       - configmaps
     verbs:
       - get
+  - apiGroups:
```

Comment:
We added new functionality in Kubernetes to support network policies https://github.com/kubernetes-sigs/kind/pull/3612/